Gemfile

```diff
@@ -96,8 +96,8 @@ gem 'webpacker', '~> 5.4'
 gem 'webpush', github: 'ClearlyClaire/webpush', ref: 'f14a4d52e201128b1b00245d11b6de80d6cfdcd9'
 gem 'webauthn', '~> 3.0'
 
-gem 'json-ld'
-gem 'json-ld-preloaded', '~> 3.2'
+gem 'json-ld', '~> 3.3.1'
+gem 'json-ld-preloaded', '~> 3.3'
 gem 'rdf-normalize', '~> 0.5'
 
 gem 'private_address_check', '~> 0.5'
```

Comment:
after merging, remind me to try and upgrade social.flipboard.com to make sure things still work there